ncm-cups/src/main/pan/components/cups/config.pan

```diff
@@ -1,8 +1,20 @@
 # ${license-info}
```

Comment:
can you use templates like `${componentconfig}` (see http://www.quattor.org/documentation/2017/02/10/components-spring17-cleaning.html#pan-cleanup)